pkg/spec/types.go

```diff
@@ -35,6 +35,7 @@ type VolumeClaim struct {
 	// k8s: io.k8s.kubernetes.pkg.api.v1.PersistentVolumeClaimSpec
 	api_v1.PersistentVolumeClaimSpec `json:",inline"`
 	// Size of persistent volume
+	// +optional
```

Comment:
Size is a required field! PersistentVolumeClaim can't be created without size.

Reply:
As per file reference, `size` conflicts with  `resources`, Hence `size` can be optional, I think so

Reply:
Reference: http://kedgeproject.org/file-reference/#persistentvolumeobject

Reply:
That is true, but one of them has to be set, either 
```
  resources:
    requests:
      storage: 500Mi
```
or 
`size: 500Mi`

Reply:
yes and no.
Exactly one has to be set either `resources.requests.storage` or `size.
One of them has to be set otherwise, PVC can't be created.

One of those has to be set. If both are set than it is conflict and we don't know which one to use. And if neither is set than its incomplete PVC definition.